### PR TITLE
Correct grammar

### PR DESCRIPTION
**Pull request by minhtaile2712**
### Issue

Closes: #

### CODE REVIEW

**Comment by stmcginnis** [comment]
Current grammar is correct, and although either wording could work, I think the existing text is slightly better.

**Comment by network-charles** [comment]
I agree with @stmcginnis.

**Comment by minhtaile2712** [comment]
The word in that position should be a verb.
`increase` is the correct form of word that should be used.
`increment` is a noun as in these dictionaries [here](https://www.oxfordlearnersdictionaries.com/definition/english/increment?q=increment) and [here](https://www.ldoceonline.com/dictionary/increment).

File: content/en/docs/concepts/workloads/pods/_index.md [hunk 1]
```diff
@@ -227,7 +227,7 @@ have some limitations:
 
 - Most of the metadata about a Pod is immutable. For example, you cannot
   change the `namespace`, `name`, `uid`, or `creationTimestamp` fields;
-  the `generation` field is unique. It only accepts updates that increment the
+  the `generation` field is unique. It only accepts updates that increase the
```
Comment (network-charles) [inline content/en/docs/concepts/workloads/pods/_index.md:230]: @minhtaile2712, an increase doesn't happen at a fixed amount, unlike an increment. You can try the demo below:

**deployment.yaml**
```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: test-increment
spec:
  replicas: 1
  selector:
    matchLabels:
      app: nginx
  template:
    metadata:
      labels:
        app: nginx
    spec:
      containers:
      - name: nginx
        image: nginx:alpine
        ports:
        - containerPort: 80
```
```bash
kubectl apply -f deployment
sleep 5
kubectl get deployments -o yaml | grep -i generation
```
You'll get the below output:
```bash
    generation: 1
    observedGeneration: 1
```

Now update the image tag and view the `generation` value:
```bash
kubectl set image deployment/cluster-exec-deployment nginx=nginx:1.20
kubectl get deployments -o yaml | grep -i generation
```

Notice the increment in the below output:
```bash
    generation: 2
    observedGeneration: 2
```
Also, according to this [dictionary](https://en.bab.la/dictionary/english/increment), an increment can be a verb in a computing context.

**Comment by sftim** [comment]
/hold

The existing text is, for technical writing, idiomatic. Let's not make this change.
Sorry to be the bearer of bad news @minhtaile2712 but I can't see any grounds to accept it. If you do see some, please update the PR description and also reply briefly in a comment.

**Comment by minhtaile2712** [comment]
After seriously researching the terms again, I admire that the existing wording is more thoughtful. You guys are correct and this should not be changed.
Thank you for your comments. I'm happy and cheerful to contribute next time.
Let's close this.

**Pull request closed without merging**